app/views/layouts/layout.ejs

```diff
@@ -30,6 +30,7 @@
             <% if (currentUser) { %>
             <li><a href="/users">Participants</a></li>
             <li><a href="/teams">Teams</a></li>
+            <li><a href="https://github.com/CodingForProduct/cfp_website/wiki">Wiki</a></li>
```

Comment:
Sure thing.   I wasn't sure if you wanted the world to have those links so I lumped them in.